src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -153,6 +153,8 @@ def load_arguments(self, _):
                    completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
                    help="name or resource id of the app service plan. Use 'appservice plan create' to get one",
                    local_context_attribute=LocalContextAttribute(name='plan_name', actions=[LocalContextAction.GET]))
+        c.argument('vnet', help="Name or resource ID of the virtual network. If there are multiple vnets of the same name across different resource groups, use vnet resource id to specify which vnet to use. If vnet name is used, by default, the vnet in the same resource group as the webapp will be used. Must be used with --subnet argument.")
```

Comment:
can you check with the VNET intergration command. - if this text needs to say regional VNET - for clarity?

Reply:
Looks like the vent-integration commands call it a "regional vnet". I'll go ahead and change it here